docs/reconciliation.md

```diff
@@ -94,6 +94,17 @@ annotations:
   eno.azure.io/deletion-strategy: orphan
 ```
 
+## Suspending Reconciliation
+
+A composition can be suspended by setting the `suspend` field in the spec to `true`.
+When a composition is suspended, the eno-reconciler will stop reconciling any related resources.
+Suspended compositions automatically infer the orphan deletion policy, preventing resources from being deleted while suspended.
```

Comment:
Let's add a note to capture the behavior of resynthesis while suspend: true. What will happen when resources in the new synthesis aren't reconciled? It looks like the synthesis will have `Synthesized != nil` but never `Reconciled !=nil`